src/main/scala/zio/actors/ActorSystem.scala

```diff
@@ -269,6 +279,17 @@ private[actors] object ActorSystemUtils {
         )
     }
 
+  def buildFinalName(parentActorName: String, actorName: String): Task[String] =
```

Comment:
This should probably be private.

Reply:
You mean private, but within the ActorSystem class?